### PR TITLE
Merge 4.3.6 into master

### PR DESCRIPTION
**Pull request by vcerenu**
This PR merge 4.3.6 version into master branch

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,23 @@
...
+
+### Added
+
+- Support to 4.3.5 Wazuh release.
+
+## Wazuh Puppet v4.3.4
+
+### Added
+
+- Support to 4.3.4 Wazuh release.
```
Comment (okynos) [inline CHANGELOG.md:20]: Put 4.5.0 above this block

File: manifests/params_manager.pp [hunk 2]
```diff
@@ -5,7 +5,11 @@
     'Linux': {
 
     # Installation
+<<<<<<< HEAD
+      $server_package_version                          = '4.3.6-1'
+=======
       $server_package_version                          = '4.5.0-1'
+>>>>>>> 69c98fe0403253e3c7f793b1c0c1a1cbccb24679
```
Comment (okynos) [inline manifests/params_manager.pp:12]: Fix conflict please

**Review by okynos — CHANGES_REQUESTED** [review]
Minor changes

File: CHANGELOG.md (hunk 1 above)
Comment (vcerenu) [reply CHANGELOG.md]: Fixed

File: manifests/params_manager.pp (hunk 2 above)
Comment (vcerenu) [reply manifests/params_manager.pp]: Fixed

**Review by okynos — APPROVED** [review]

**Pull request merged**